document/CoreFormat.mdk

```diff
@@ -1743,13 +1729,138 @@ A `codeFlow` object may contain a property named `tags`, whose value is a tag se
+
+~ IsoNote
+It is possible that the analysis tool will not have location information for every frame in the call stack.
+This might happen if, for example, application code for which location information is available calls into operating system code
+for which location information is not available, which in turn calls back into application code.
+~
+
+### `tags` property { #stack-tags }
+
+A `stack` object may contain a property named `tags`, whose value is a tag set.
+
+### `properties` property { #stack-properties }
+
+A `stack` object may contain a property named `properties`, whose value is a property bag.
+This allows tools to include information about the stack that is not explicitly specified in the SARIF format.
+
+## `stackFrame` object { #stackFrame }
+
+### General { #stackFrame-General }
+
+A `stackFrame` object describes a single stack frame within a call stack (&sect;[#stack]).
+
+### `message` property { #stackFrame-message }
+
+A `stackFrame` object may contain a property named `message` whose value is a string
+containing a message relevant to this stack frame.
+
+### `uri` property { #stackFrame-uri }
+
+A `stackFrame` object may contain a property named `uri` whose value is a string
+containing the URI of the source code file to which this stack frame refers.
+
+~ IsoNote
+
+The `uri` property will be absent if the analysis tool does not have information
+that maps execution addresses to source code files.
+~
+
+### `line` property { #stackFrame-line }
+
+A `stackFrame` object may contain a property named `line` whose value is an integer
+containing the line number within the file specified by `uri` (&sect;[#stackFrame-uri])
+to which this stack frame refers.
+
+If the `uri` property is absent, the `line` property shall be absent.
+
+~ IsoNote
+The `line` and `column` (&sect;[#stackFrame-column]) properties will be absent if the
+analysis tool does not have information that maps execution addresses to source code lines.
+~
+
+### `column` property { #stackFrame-column }
+
+A `stackFrame` object may contain a property named `column` whose value is an integer
+representing the column number within the line specified by `line` (&sect;[#stackFrame-line])
+to which this stack frame refers.
+
+If the `line` property is absent, the `column` property shall be absent.
+
+### `module` property { #stackFrame-module }
+
+A `stackFrame` object may contain a property named `module` whose value is a string
+containing the name of the module that contains the location to which this stack frame refers.
+
+### `fullyQualifiedLogicalName` property { #stackFrame-fullyQualifiedLogicalName }
+
+A `stackFrame` object may contain a property named `fullyQualifiedLogicalName` whose value
+is a string containing the fully qualified name of the method to which this stack frame refers.
+See &sect;[#location-fullyQualifiedLogicalName] for examples.
+
+~ IsoNote
+The `fullyQualifiedLogicalName` property will be absent in the analysis tool does not
```

Comment:
there is a typo in this statement s/be 'if the analysis tool' not 'in the analysis tool'. the statement is also not correct. a managed stack frame has no symbol information but constructs a fully qualified name (due to the discoverability of IL).